api/v1/postgres_types.go

```diff
@@ -481,3 +493,138 @@ func deleteIfEmpty(json map[string]interface{}, key string) {
 func init() {
 	SchemeBuilder.Register(&Postgres{}, &PostgresList{})
 }
+
+func (p *Postgres) buildAdditionalVolumes(c *corev1.ConfigMap) []zalando.AdditionalVolume {
+	if c == nil {
+		// abort if the global configmap is not there
+		return nil
+	}
+
+	return []zalando.AdditionalVolume{
+		{
```

Comment:
Since all info of `[]zalando.AdditionalVolume` is known at compile time, we can shift this part of logic to a `var`. For example:
```
var additionalVolumes = []zalando.AdditionalVolume{...}
```

Reply:
That is indeed true, good point.

Reply:
Since it's more of a *nice to have* thing and we have another experimental branch (#157) for this part of logic, I would resolve the conversation here.